src/plugins/data/public/ui/index.ts

```diff
@@ -46,6 +39,10 @@ export {
   QueryEditorExtensions,
   QueryEditorExtensionDependencies,
   QueryEditorExtensionConfig,
+  createEditor,
+  DefaultInput,
```

Comment:
instead of exporting the type, export the createEditor from the ui service, and then you can also export the default input and single line input